Dockerfile

```diff
@@ -57,12 +57,12 @@ RUN git clone https://github.com/libimobiledevice/libplist \
 
 # Installing MVT
 # --------------
-RUN pip3 install mvt
+RUN pip3 install git+https://github.com/mvt-project/mvt.git@main
```

Comment:
Ideally I think we should be copying in from whatever's in the locally checked out directory, but this seems like an improvement regardless

Reply:
I could update it for that,  though I wonder would it make more sense for you to put it up on docker hub?  It'd cut down on the steps a little to get the container up and running.

Reply:
I would be in favour of us publishing an official docker image